### PR TITLE
Added missing stringToObjectId / objectIDToString methods

### PR DESCRIPTION
**Pull request by tobydeh**
Added missing stringToObjectId / objectIDToString methods to mongoose adapter

### CODE REVIEW

**Comment by AndreMaz** [comment]
Thanks @tobydeh 😄  Can you also add the unit tests? 
The tests can be equal to those present in mongo adapter
https://github.com/moleculerjs/moleculer-db/blob/master/packages/moleculer-db-adapter-mongo/test/unit/index.spec.js

File: packages/moleculer-db-adapter-mongoose/src/index.js [hunk 1]
```diff
@@ -383,6 +383,29 @@ class MongooseDbAdapter {
...
+	/**
+	 * Convert hex string to ObjectID
+	 *
+	 * @param {String} id
+	 * @returns {ObjectID}
+	 *
+	 * @memberof MongooseDbAdapter
+	 */
+	stringToObjectId (id) {
+    return new mongoose.Schema.Types.ObjectId(id);
```
Comment (AndreMaz) [inline packages/moleculer-db-adapter-mongoose/src/index.js]: I think that we should check if the `id` is valid. Just as done in mongo adapter
https://github.com/moleculerjs/moleculer-db/blob/81919e05a67ac9c973f4743517bcee7a6013aabf/packages/moleculer-db-adapter-mongo/src/index.js#L360-L365

I think that `isValid` method is avalidable as `mongoose.Types.ObjectId.isValid(id)`

**Comment by tobydeh** [comment]
Done

File: packages/moleculer-db-adapter-mongoose/test/unit/index.spec.js [hunk 2]
```diff
@@ -166,6 +166,31 @@ describe("Test MongooseStoreAdapter", () => {
 			});
 		});
 
+		it("call stringToObjectID", () => {
+			mongoose.Types.ObjectId.isValid = jest.fn(() => true);
+			mongoose.Schema.Types.ObjectId= jest.fn();
```
Comment (AndreMaz) [inline packages/moleculer-db-adapter-mongoose/test/unit/index.spec.js]: Can you please lint the code and fix the warnings?

Comment (tobydeh) [reply packages/moleculer-db-adapter-mongoose/test/unit/index.spec.js]: The only eslint errors i see relate to the other adapters.

**Comment by AndreMaz** [comment]
Sorry for the delay @tobydeh . Thanks for the PR 👍

**Pull request merged**